web-ng/src/app/components/drawing-tools/drawing-tools.component.ts

```diff
@@ -33,16 +39,27 @@ import { DomSanitizer, SafeUrl } from '@angular/platform-browser';
   styleUrls: ['./drawing-tools.component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class DrawingToolsComponent {
+export class DrawingToolsComponent implements OnInit, OnDestroy {
+  private subscription: Subscription = new Subscription();
   pointValue = 'point';
   polygonValue = 'polygon';
   selectedValue = '';
   private lastSelectedValue = '';
   selectedLayerId = '';
   readonly layers$: Observable<List<Layer>>;
   readonly lang: string;
+  private black = '#202225';
```

Comment:
If we can't get this from styles, perhaps define it as at file level as `readonly const BLACK = ..;`. Same for GREEN.

Reply:
Done.